common/workunit/workunit.cpp

```diff
@@ -8748,9 +8748,12 @@ bool CLocalWorkUnit::getSummary(SummaryType type, SummaryMap &map) const
     StringArray s;
     {
         CriticalBlock block(crit);
-        const char *list = p->queryProp(xpath);
-        if (!list)
+        IPropertyTree * match = p->queryPropTree(xpath);
+        if (!match)
             return false;
```

Comment:
does it make sense to return false if no xpath match ,but true (below) if !list ?
maybe it does - false means bad format (given other return false)

Reply:
false means this workunit did not record the information.  true means it recorded it, but it was empty.
I will add a comment to clarify.